### DISCUSSION TITLE
py files?

### DISCUSSION THREAD

**Discussion by mscroggs** [jorgensd/dolfinx_docs#3] [General]
I think we should use py files with `#-` comments that allow them to be converted to notebooks easily instead of ipynb files. What do you think?

**Comment by mscroggs** [comment]
I'm going to make a PR with this conversion for the mesh page to show what I mean

**Reply by jorgensd** [reply]
I know what you mean. 
It is what I use in DOLFINx tutorial.
However, for cases where we want to highlight MPI usage (i.e. running multiple processers), we cannot convert it directly (i think). Feel free to try though (ipyparallel is new to me)
